src/app/hearing-part/models/hearing-part.viewmodel.ts

```diff
@@ -1,14 +1,22 @@
 import * as moment from 'moment'
-import { Session } from '../../sessions/models/session.model';
+import { Note } from '../../notes/models/note.model';
+import { Priority } from './priority-model';
+import { Judge } from '../../judges/models/judge.model';
 
-export interface HearingPartViewmodel {
+export interface HearingPartViewModel {
     id: string;
-    session: Session;
+    session: string;
```

Comment:
later on, I would suggest changing the field name, if it's not a full session anymore (is it id?)